client/modules/food/components/packing/Packages.js

```diff
@@ -39,14 +53,37 @@ class Packages extends Component {
 
   renderSizePerPageDropDown = () => <SizePerPageDropDown variation='dropup'/>
 
-  getActionButtons = (_, foodPackage) =>
-    <div>
+  getActionButtons = (_, foodPackage) => {
+    return (
       <Button bsStyle="danger"
         onClick={() => this.props.unpack(foodPackage._id)}
+        disabled={foodPackage.status === 'Received'}
       ><i className="fa fa-undo" /> Unpack</Button>
-    </div>
-
+    )
+  }
+    
+  getPackageButton = (_, foodPackage) => {
```

Comment:
Why not do this in `getActionButtons` above? Then they can be in the same table column